packages/components/src/tabs/styles.ts

```diff
@@ -120,6 +120,12 @@ export const Tab = styled( Ariakit.Tab )`
 			opacity: 1;
 		}
 	}
+
+	[aria-orientation='vertical'] & {
+		min-height: ${ space(
+			10
+		) }; // Avoid fixed height to allow for long strings that go in multiple lines.
+	}
```

Comment:
Can we wrap this in a media query? For touch devices the larger 48px click area is helpful.

Reply:
> For touch devices the larger 48px click area is helpful.

FWIW, the 40px size (the new default height for basic control components) was chosen specifically to be touch-friendly, so I don't think we need to upsize on touch devices.

Reply:
I added a breakpoint. Unfortunately we can't use the mixin in typescript.